src/docfx/lib/HttpRequestUtility.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Net.Http;
+
+namespace Microsoft.Docs.Build
+{
+    internal static class HttpRequestUtility
```

Comment:
not in use?

Reply:
it is an extension of `HttpRequestMessage `, used in `CredentialHandler.cs`.